content/get-involved.md

```diff
@@ -4,3 +4,42 @@ Title: "Get Involved"
 
 Are you interested in participating in the OpenTelemetry community? Read on to learn more
 about how to get involved.
+
+# Special Interest Groups
+We organize the community into Special Interest Groups (SIGs) in order to improve our workflow and more easily manage a community project.
+
+[Cross-language Specification] (https://github.com/open-telemetry/community#cross-language-specification)
```

Comment:
These links are not proper Markdown. Remove the space between the bracket and the parenthesis.

Reply:
fixed, i think! sorry about that.